node/src/snark_pool/candidate/snark_pool_candidate_actions.rs

```diff
@@ -49,6 +49,7 @@ pub enum SnarkPoolCandidateAction {
     WorkVerifyError {
         peer_id: PeerId,
         verify_id: SnarkWorkVerifyId,
+        batch: Vec<Snark>,
```

Comment:
@0xMimir what about just passing the job ids here? Seems the whole snark is not actually needed